CalibMuon/RPCCalibration/python/l1Muon_EventContent_cff.py

```diff
@@ -2,9 +2,12 @@
 
```

Comment:
I didn't find this file included in standard workflows with RECO step.
We have other digi producers (simMuon\* ; hltMuon\*).

So, changes in this file should probably not be made.